src/java/org/apache/cassandra/index/internal/CassandraIndex.java

```diff
@@ -68,6 +68,8 @@
  */
 public abstract class CassandraIndex implements Index
 {
+    public static final String NAME = "cassandra";
```

Comment:
The name "cassandra" here isn't perfect I suppose, but it's trying to a.) align w/ the class name at least somewhat and b.) connote that this is the "traditional cassandra hidden table" index.

Reply:
This is the point I most want to comment on this patch~~~ Because  cassandra vs SAI cannot be aligned conceptually in my mind.  Even native2i is better than cassandra

Reply:
I’d really like to call it “legacy” to reflect my belief that we will eventually treat SAI as the default 2i implementation.

Reply:
legacy is also one of the potential options I think

Reply:
Posted an informal poll in ASF Slack's #cassandra-sai chat

Reply:
@adelapena chimed in with "legacy_local_table", which I like a lot. I'm thinking about proceeding w/ that.